restaurants/views.py

```diff
@@ -69,3 +72,30 @@ def get(self, request, restaurant_id):
 
         except Restaurant.DoesNotExist:
             return JsonResponse({"message":"RESTAURANT_NOT_EXIST"}, status=404)        
+
+class RestaurantReviewView(View):
+    @ConfirmUser
+    def post(self, request, restaurant_id):
+        try:
+            data       = json.loads(request.body)
+            restaurant = Restaurant.objects.get(id=restaurant_id)
+            content    = data["content"]
+            rating     = data["rating"]
+
+            Review.objects.create(
+                user       = request.user,
+                restaurant = restaurant,
+                content    = content,
+                rating     = rating
+            )
+
+            return JsonResponse({"message":"success"}, status=201)
```

Comment:
return하는 message 형식도 맞춰주세요~
```suggestion
            return JsonResponse({"message":"SUCCESS"}, status=201)
```

Reply:
마이너한 리뷰에요~

Reply:
해당 사항 마이너하게 수정했습니다!